audio/engine.asm

```diff
@@ -2117,19 +2153,18 @@ GetMusicByte:
 	ld hl, CHANNEL_MUSIC_BANK
 	add hl, bc
 	ld a, [hl]
-	; get byte
-	call _LoadMusicByte ; load data into wCurMusicByte
-	inc de ; advance to next byte for next time this is called
+	; load byte into wCurMusicByte
+	call _LoadMusicByte
+	; advance to next byte for next time this is called and
```

Comment:
By moving this comment outside of the line where it was, you have made the flow more obscure. Before, it was clear that `inc de` was simply incrementing the pointer for the next time, and that the following code was updating the channel data address. Now that's merged into a single comment, still broken down into two lines for some unexplainable reason.